.evergreen/stop-orchestration.sh

```diff
@@ -1,14 +1,20 @@
 #!/usr/bin/env bash
 set -o errexit  # Exit the script with error if any of the commands fail
 
-cd "$MONGO_ORCHESTRATION_HOME"
+SCRIPT_DIR=$(dirname ${BASH_SOURCE:-$0})
```

Comment:
```suggestion
SCRIPT_DIR=$(dirname "${BASH_SOURCE:-"$0"}")
```
ShellCheck SC2086: Double quote to prevent globbing and word splitting.

Also applies to other variable expansions in this PR.